File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 5/PRAK505-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
void biodata(int tahunlahir, char namaku[20], char asal[15]){
    int tahun_sekarang = 2020;
    printf("Perkenalkan Nama Saya : %s\n",namaku);
    printf("Umur saya : %d\n",tahun_sekarang - tahunlahir);
    printf("Saya Adalah Angkatan : %d\n",tahun_sekarang);
    printf("Asal Saya dari : %s\n",asal);
}
    int main()
    {
    int tahunlahir;
    char namaku[20],asal[15];
    scanf(" %d",&tahunlahir);
    scanf(" %[^\n]s",&namaku);
    scanf(" %[^\n]s",&asal);
    biodata(tahunlahir,namaku,asal);
    return 0;
}

File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 5/PRAK505-2210817210014-MuhammadQoharyRawidafany.py
def biodata(tahunlahir,namaku,asal) :
    tahunlahir = int (tahunlahir)
    tahun_sekarang = 2020
    print("Perkenalkan Nama Saya : ",namaku)
    print("Umur saya : ",tahun_sekarang - tahunlahir)
    print("Saya Adalah Angkatan : ",tahun_sekarang)
    print("Asal Saya dari : ",asal)
tahunlahir = int (input(""))
namaku = input("")
asal = input("")
biodata(tahunlahir,namaku,asal)